designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,49 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
```

Comment:
I think we should rephrase this section to not refer to it being a PR and just explain where templates live. Ultimately it will be the documentation for the section. In the future, you could add certain notes to the PR comment if needed.